JetStreamDriver.js

```diff
@@ -813,6 +793,26 @@ class Benchmark {
         return code;
     }
 
+    prepareBrowserUI() {
```

Comment:
Please excuse another nit: I find it a bit confusing that (1) this by itself doesn't actually prepare or change any UI, since it's stateless and just returning text, and (2) that it has the same name as the `Driver.prepareBrowserUI` which _does_ actually update the UI and has a different signature. Maybe rename this one (and the overridden variant in GroupedBenchmark`) to `getResultDivText` or something (whatever it is, should be different from the Driver method, and make clear that this returns something, not update on its own).